src/droplet.F

```diff
@@ -2282,12 +2286,20 @@ subroutine LV_solver(diffnorm,dt_nondim,taup_scale,rt_start,rt_zeros,flag,prsval
 
         error = 1.0e-8
 
-        I = reshape((/1, 0, 0, 1/),(/2,2/))
+        
+        !JMD-FIXME generates an internal compiler eerror
+        ! I = reshape((/1, 0, 0, 1/),(/2,2/))
+        I(1,1)=1.0
+        I(2,1)=0.0
+        I(1,2)=0.0
+        I(2,2)=1.0
         iterations = 0
         flag = 0
         v1(1) = rt_start(1)
         v1(2) = rt_start(2)
-        fv2 = (/1., 1./)
+        !JMD-FIXME generates an internal compiler error
+        ! fv2 = (/1., 1./)
+        fv2(:) = 1.0
```

Comment:
Will it be better to write `fv2(1) = 1.0` and  `fv2(2) = 1.0`?

Reply:
Just updated the code to eliminate the array syntax.